tests/generators/schema/website-test.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Yoast\WP\SEO\Tests\Generators\Schema;
+
+use Mockery;
+use Yoast\WP\SEO\Helpers\Options_Helper;
+use Yoast\WP\SEO\Helpers\Schema\HTML_Helper;
+use Yoast\WP\SEO\Helpers\Schema\ID_Helper;
+use Yoast\WP\SEO\Presentations\Generators\Schema\Website;
+use Yoast\WP\SEO\Tests\Mocks\Meta_Tags_Context;
+use Yoast\WP\SEO\Tests\TestCase;
+
+/**
+ * Class Website_Test
+ *
+ * @group generators
+ * @group schema
+ *
+ * @coversDefaultClass \Yoast\WP\SEO\Presentations\Generators\Schema\Website
+ */
+class Website_Test extends TestCase {
+
+	/**
+	 * The instance to test.
+	 *
+	 * @var Website
+	 */
+	private $instance;
+
+	/**
+	 * The options helper.
+	 *
+	 * @var Options_Helper|Mockery\MockInterface
+	 */
+	private $options;
+
+	/**
+	 * The HTML helper.
+	 *
+	 * @var HTML_Helper|Mockery\MockInterface
+	 */
+	private $html;
+
+	/**
+	 * The ID helper.
+	 *
+	 * @var ID_Helper|Mockery\MockInterface
+	 */
+	private $id;
+
+	/**
+	 * The meta tags context object.
+	 *
+	 * @var Meta_Tags_Context
+	 */
+	private $meta_tags_context;
+
+	/**
+	 * Sets up the tests.
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		$this->options = Mockery::mock( Options_Helper::class );
+		$this->html    = Mockery::mock( HTML_Helper::class );
+		$this->id      = new ID_Helper();
```

Comment:
The id helper is only using inside this method, I wouldn't make it a class property